### DISCUSSION TITLE
Have you tried this published to a server ?

### DISCUSSION THREAD

**Discussion by bbbkada** [ashstrahle/PowerShellBlazor#4] [General]
I can get it to run in Visual Code, but not on IIS.
I tried running the appool with the same account as when successfully running it in VS.
Still nothing returned from the powershell script, only the Begin/Complete strings returned.

**Comment by bbbkada** [comment]
When I run single commands, it works. But not with .ps1
Tried setting Execution-policy for the appool user, without success.
Any ideas ?

**Reply by ashstrahle** [reply]
Hi @bbbkada. I've certainly had this code working on IIS in another lifetime. That said, I no longer have access to Windows server architecture, and primarily run on linux docker instances. 

A distant memory of this was having  IIS run under a service account, however I cannot be certain. Sorry I can't be more explicit.
